st2common/st2common/models/db/actionalias.py

```diff
@@ -65,6 +65,9 @@ class ActionAliasDB(stormbase.StormBaseDB, stormbase.ContentPackResourceMixin,
     result = me.DictField(
         help_text='Parameters pertaining to the execution result message.'
     )
+    extras = me.DictField(
+        help_text='Additional parameters not covered in the schema.'
```

Comment:
Maybe clarify that those can be adapter specific attributes and things like that.